api.md

```diff
@@ -288,11 +288,13 @@ to how you handle constraints. A hypertable can contain foreign keys to normal S
 columns, but the reverse is not allowed. UNIQUE and PRIMARY constraints must include the
 partitioning key.
 
->:WARNING: Running `create_hypertable` with data migration on a table with foreign key contraints to other tables 
-might result in a deadlock if parallel transactions simultaneously try to insert data into the table and 
-the referenced tables. This can result in aborting `create_hypertable`. The deadlock can be prevented by 
-manually obtaining `SHARE ROW EXCLUSIVE` lock on the referenced tables  before calling `create_hypertable` 
-in the same transaction.
+>:WARNING: Running `create_hypertable` with the data migration on a table with foreign key contraints 
+might result in a deadlock. This is likely to happen if parallel transactions simultaneously try 
+to insert data into tables referenced from the converting table and into the converting table itself. 
```

Comment:
this is confusing.  How about `insert data into the foreign tables referenced by the hypertable or into the hypertable itself.`

Reply:
> foreign tables

Unfortunately others are confused with term `foreign table`, which means remote table accessible through foreign data wrapper.

Reply:
How about "insert data into the foreign tables (i.e tables referenced by the hypertable's foreign key constraints)".  This clarifies what is meant by foreign tables here.

Reply:
I improved the sentence. [It](https://github.com/timescale/docs.timescale.com-content/pull/290/files#diff-00e71df22262087fd8ad820708997657R292-R294) should be better, but I am not sure if it is enough.